src/javascript/query-explorer/index.js

```diff
@@ -149,6 +151,14 @@ function handleDataChartSuccess(data) {
       response: data.response
     }
   });
+
+  ga('send', {
+    hitType: 'event',
+    eventCategory: 'query',
+    eventAction: 'submit:success',
+    eventLabel: '(not set)',
```

Comment:
Since you use the error message for the label when there is an error, how about using the code and a message for a success, instead of (not set). E.g. (200) OK.